crates/re_viewer/src/ui/recordings_panel.rs

```diff
@@ -64,32 +64,29 @@ fn loading_receivers_ui(
     let mut any_shown = false;
 
     for source in rx.sources() {
-        let (always_show, string) = match source.as_ref() {
-            SmartChannelSource::File(path) => (false, format!("Loading {}…", path.display())),
+        let string = match source.as_ref() {
+            SmartChannelSource::File(path) => format!("Loading {}…", path.display()),
 
-            SmartChannelSource::RrdHttpStream { url } => (false, format!("Loading {url}…")),
+            SmartChannelSource::RrdHttpStream { url } => format!("Loading {url}…"),
 
-            SmartChannelSource::RrdWebEventListener => {
-                (false, "Waiting on Web Event Listener…".to_owned())
-            }
+            SmartChannelSource::RrdWebEventListener => "Waiting on Web Event Listener…".to_owned(),
```

Comment:
Although we (or at least some of us :sweat_smile: ) know what this is, this probably isn't a good name for user-visible text. Practically speaking today this would be something like "Waiting on data from notebook"